chat-plugins/poll.js

```diff
@@ -56,32 +58,38 @@ class Poll {
 	}
 
 	generateVotes(num) {
-		let output = '<div class="infobox"><details><summary style="margin: 2px 0 5px 0"><span style="border:1px solid #6A6;color:#484;border-radius:4px;padding:0 3px"><i class="fa fa-bar-chart"></i> Poll-' + this.pollArray[num].pollNum + '</span> <strong style="font-size:11pt">' + this.getQuestionMarkup(num) + '</strong></summary>';
+		let output = `<div class="infobox"><details><summary style="margin: 2px 0 5px 0"><span style="border:1px solid #6A6;color:#484;border-radius:4px;padding:0 3px"><i class="fa fa-bar-chart"></i> Poll-${this.pollArray[num].pollNum}</span> <strong style="font-size:11pt">${this.getQuestionMarkup(num)}</strong><img src="https://pldh.net/media/pokecons_action/196.gif" width="32" height="32"></summary>`;
```

Comment:
Use a `<psicon>` for the icon?

EX: `<psicon pokemon="espeon">`
Its a custom HTML tag (and you dont need to close it) thats parsed by the client.